flutter_app/lib/main.dart

```diff
@@ -1,18 +1,21 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'Stats.dart';
+import 'deviceVars.dart';
+//import 'package:charts_flutter/flutter.dart'; // this line Imported
```

Comment:
Don't forget to remove ghost code as it bloats up your project

Reply:
will check to remove if issue doesn't get resolved

Reply:
resolved